package.json

```diff
@@ -30,32 +30,31 @@
     "build:theme-designer": "yarn workspace @twilio-paste/theme-designer build",
     "build:props": "lerna run build:props --ignore @twilio-paste/website --concurrency 2",
     "pre-push": "concurrently \"yarn:lint\" \"yarn:test\" \"yarn:prettier\" \"yarn:type-check\"",
-    "release": "yarn lerna publish -m 'chore(release): publish' --conventional-commits --canary --preid beta --dist-tag next",
-    "release:stable": "yarn lerna publish -m 'chore(release): publish' --conventional-commits",
-    "release:stable:ci": "yarn lerna publish -y --loglevel verbose -m 'chore(release): publish' --conventional-commits",
-    "release:stable:npm-failed": "yarn lerna publish from-package -y --loglevel verbose -m 'chore(release): publish' --conventional-commits",
+    "prerelease": "yarn build && yarn lint && yarn test",
+    "release": "yarn changeset publish",
```

Comment:
Can remove the `changeset` script and replace this with `"release": "changeset publish"`

Reply:
I think we need the `changeset` script so we can generate a changeset for each PR, but I might be missing something there.

Reply:
Good point.